CrewChiefV4/speech_recognition_config.txt

```diff
@@ -161,7 +162,14 @@ ON = on
 # 's added to a name or number to indicate possession - "what is bob's best lap", "what's position 12's last lap"
 POSSESSIVE = 's
 
-
+# used for fuel calculations 
+LAP = lap
+LAPS = laps
+MINUTE = minute
+MINUTES = minutes
+HOUR = hour
+HOURS = hours
+AND = and
```

Comment:
i'd be tempted to remove AND here, so the app only supports requests for 0 to 59 minutes, or 1 to 24 hours. And nothing in between (no "one hour and twenty minutes")

Reply:
i get your point here but i think we are going to maby need numbers up to 90 then, maby we can go with only having 70,80 and 90

Reply:
hmm number parser wont work to well if its missing something in betwene., i'll just add its no big deal.